releasenotes/notes/collect-recommended-k8s-tags-095651035a2030a7.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Tagger now collects recommended "app.kubernetes.io" Kubernetes labels as
```

Comment:
nit: we don't expect users to know about tagger as an agent component, you can replace it by "The  Agent"